src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkOutOfDateException.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.appengine.cloudsdk;
+
+/**
+ * The Cloud SDK that was found is too old (generally before 131.0).
+ */
+public class CloudSdkOutOfDateException extends CloudSdkNotFoundException {
```

Comment:
I think we're fudging things a bit to say that an incompatible version is 'not found'.